README.md

```diff
@@ -207,6 +207,11 @@ pip install nvidia-cublas-cu12==12.4.5.8  # for H20
 ```
 Note: We test following examples on hardware of NVIDIA H800-80GB with CUDA 12.2. Loading inclusionAI/Ming-Lite-Omni in bfloat16 takes about 40890MB memory.
 
+You can also initialize the environment by building the docker image:
```

Comment:
1. put under a '# Docker' section 
2. Provide a step-by-step guide for docker environment setup